frontend/src/pages/Profile.jsx

```diff
@@ -61,26 +69,48 @@ class Profile extends Component {
         )
     }
 
+    componentDidMount() {
+        this.getProfile();
+    }
+
+    getProfile() {
+        var urlPath = "/api/author/"
+        var authorId = store.getState().loginReducers.userId.split("thor/");
+        const path = urlPath + authorId[1], requireAuth = true;
+        HTTPFetchUtil.getRequest(path, requireAuth)
+        .then((httpResponse) => {
+            if (httpResponse.status === 200) {
+                httpResponse.json().then((results) => {
+                    this.setState( {
+                        profiledata: results
+                    })
+                })
```

Comment:
In the case that the promise returned from .json() gets rejected for whatever reason, it's good practice to have a .catch block to handle the rejected promises.

Reply:
this is our fight